sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/StorageAccountInfo.java

```diff
@@ -36,4 +51,11 @@ public SkuName getSkuName() {
     public AccountKind getAccountKind() {
         return accountKind;
     }
+
+    /**
+     * @return whether hierarchical namespace is enabled on the account.
+     */
+    public Boolean isHnsEnabled() {
```

Comment:
In the .net client this method is named `IsHierarchicalNamespaceEnabled`: 
https://learn.microsoft.com/en-us/dotnet/api/azure.storage.blobs.models.accountinfo.ishierarchicalnamespaceenabled?view=azure-dotnet#azure-storage-blobs-models-accountinfo-ishierarchicalnamespaceenabled

BTW, I only figgured out how to do this by asking ChatGPT and it gave me code that assumed a method named `IsHierarchicalNamespaceEnabled` existed in this class 😄